features/db-columns.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Display information about a given table.
+
+  Scenario: Display information about the wp_posts table
```

Comment:
Schema might not be the same for older WordPress versions. A minimum version should be added.

Reply:
I think the last change to the database schema was for WP 4.2, so adding the following should work: `@require-wp-4.2`